### DISCUSSION TITLE
ansible role running error

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1666] [Q&A] [answered]
hello
 when i run ansible role for frontend backend its running only expense.yml file. not running frontend backend services

![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/2a6909bc-cfce-4396-b936-e92b7a50f6bb)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/47cb869e-4a59-4553-9b96-06b088bce412)

**Comment by sreedharm07** [comment]
@aambadkar 
Inside tasks it will only run main.yml file but in your repos there is no main.yml file

Rename  all files inside tasks of frontend,backend to main.yml

**Reply by aambadkar** [reply]
ok i updated but having new error again
![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/d4d2c844-47e4-469c-9c9c-0337015c6b41)

**Reply by sreedharm07** [reply]
@aambadkar 
Check copy module in ansible  in documentation once 
Remote_src was missing please add that it will run

**Reply by aambadkar** [reply]
yes its solved it was typing miske

above problem is solved.
now everything is going perfectlt but when i run the app expenses not adding 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/5701569c-e25d-4449-9d27-8e2208615d8b)

**Reply by sreedharm07** [reply]
check service files line by line 
frontend.service , backend.service files

**Reply by aambadkar** [reply]
still not solved

**Reply by sreedharm07** [reply]
your backend was not in active configure properly it will works
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/11e06279-75ed-4975-92f8-232c6c22f2c1)

**Reply by aambadkar** [reply]
what should i do?

**Reply by sreedharm07** [reply]
did your ansible scripts runned without errors?
if any error happening in ansible script at any point do modify that module and make your ansible scripts run without any error 
after that check once again service files from documentation 

after that if you get same error please ping, i will check and will try to find out why error is happening

**Reply by aambadkar** [reply]
yes. ansible script running without any errors
also i cross checked service files from documentation. everything looks like perfect
but still not adding expenses

**Reply by sreedharm07** [reply]
ok ping your workstation ip

**Reply by aambadkar** [reply]
workstation : 3.81.217.241
frontend : 3.90.154.85

**Reply by sreedharm07** [reply]
missing port number here add it (8080)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/077ec1bb-bae9-4040-9d81-ee3a45ec2a31)

**Reply by aambadkar** [reply]
it is like that?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/88dae478-c95e-4cd3-8282-3bd25339c926)

i think its not valid i passed there parameter from parameter store

**Reply by sreedharm07** [reply, marked as answer]
you are not getting the point.
if you passed parameters why its not comming in server? 
check the parameter value once

**Reply by aambadkar** [reply]
now its working 

thank you
